plugins/dependencies/src/main/kotlin/dependencies/Testing.kt

```diff
@@ -102,6 +104,62 @@ object Testing {
         }
     }
 
+    /**
+     * Powerful, elegant and flexible test framework for Kotlin
+     *
+     * GitHub page: [kotest/kotest](https://github.com/kotest/kotest)
+     */
+    object Kotest {
+        private const val artifactBase = "io.kotest:kotest"
+
+        val runner = Runner
+        val plugins = Plugins
+        val extensions = Extensions
+        val assertions = Assertions
+
+        const val core = "$artifactBase-core:_"
+        const val property = "$artifactBase-property:_"
+        const val propertyArrow = "$artifactBase-property-arrow:_"    
+
+        object Runner {
+            private const val artifactPrefix = "$artifactBase-runner"
+
+            const val junit4 = "$artifactPrefix-junit4:_"
+            const val junit5 = "$artifactPrefix-junit5:_"
+        }
+
+        object Plugins {
+            private const val artifactPrefix = "$artifactBase-plugins"
+
+            const val piTest = "$artifactPrefix-pitest:_"
+        }
+
+        object Extensions {
+            private const val artifactPrefix = "$artifactBase-extensions"
+
+            const val spring = "$artifactPrefix-spring:_"
+            const val koin = "$artifactPrefix-koin:_"
+            const val allure = "$artifactPrefix-allure:_"
+            const val testContainers = "$artifactPrefix-testcontainers:_"
+            const val http = "$artifactPrefix-http:_"
+            const val mockserver = "$artifactPrefix-mockserver:_"
```

Comment:
If we have `piTest` and `testContainers`, shouldn't we have `mockServer` here instead of `mockserver`, for case consistency?

Reply:
I suppose I thought of mockserver as a single word but please feel free to adjust anything to match the style of this project.